### ISSUE TITLE
Lots of spaces in the plugin output of HyperV checks

### ISSUE THREAD

**Issue by K0nne** [Icinga/icinga-powershell-hyperv#28]
Hello,

while copy&pasting the plugin output for #27 (see HyperV Snapshots & HyperV VM Health) it came to my attention that the plugin output sometimes contains huge amounts of spaces. Is there a way to trim them for the sake of readability?

**Comment by LordHepipud** [comment]
Thank you for the issue. The bigger problem here is the exception which should never be printed in this way by a plugin and return a catched Framework exception instead. We will have a look on this

**Comment by LordHepipud** [comment]
This should be resolved by Icinga for Windows v1.5.0 and the new execution of checks:

```powershell
powershell -C { Use-Icinga -Minimal; Exit-IcingaExecutePlugin <plugin command> <plugin args>
```

Exception handling is done a lot better and will print proper output including the stack traces.
